haproxy/Dockerfile

```diff
@@ -0,0 +1,77 @@
+#FROM ubuntu:jammy-20230425
+FROM ubuntu:focal-20230412 AS build
```

Comment:
Please use the following ubuntu base image.
ghcr.io/cybozu/ubuntu:20.04

Reply:
I change "ghcr.io/cybozu/ubuntu:20.04" from "ubuntu:focal-2023041"